extensions/amp-next-page/0.1/amp-next-page.js

```diff
@@ -286,107 +86,24 @@ export class AmpNextPage extends AMP.BaseElement {
     user().assert(separatorElements.length <= 1,
         `${TAG} should contain at most one <div separator> child`);
 
+    let separator;
     if (separatorElements.length == 1) {
```

Comment:
This can be simplified to eliminate the `if`/`else` branching

```js
const separator = separatorElements[0] || this.win.document.createElement('div')

Reply:
Done.